lib/optimize/CommonsChunkPlugin.js

```diff
@@ -286,7 +286,7 @@ Take a look at the "name"/"names" or async/children option.`);
 
 		// count how many chunks contain a module
 		const commonModulesToCountMap = usedChunks.reduce((map, chunk) => {
-			for(const module of chunk.modulesIterable) {
+			for(const module of chunk.modules) {
```

Comment:
This change (and PR) is not correct. `this.modules` is a deprecated getter, while `modulesIterable` simply references the internal `this._modules`, which is a `SortableSet` that can be iterated properly.